dev/Dockerfile.base

```diff
@@ -62,28 +67,23 @@ RUN set -ex; \
 ENV PATH="/venv/bin:${PATH}" \
     VIRTUAL_ENV="/venv"
 
-COPY ./requirements/requirements.common.txt /tmp/requirements.txt
```

Comment:
This is related to https://issues.redhat.com/browse/AAH-1685 however that issue is not completely fixed yet until we confirm with installer teams they are not using this file.